stl/inc/type_traits

```diff
@@ -1854,7 +1857,8 @@ template <class _Ty, class _Uty, class = void>
 struct _Refwrap_has_ctor_from : false_type {};
 
 template <class _Ty, class _Uty>
-struct _Refwrap_has_ctor_from<_Ty, _Uty, void_t<decltype(_Refwrap_ctor_fun<_Ty>(_STD declval<_Uty>()))>> : true_type {};
+struct _Refwrap_has_ctor_from<_Ty, _Uty, void_t<decltype(_STD _Refwrap_ctor_fun<_Ty>(_STD declval<_Uty>()))>>
+    : true_type {}; // qualified: avoid ADL, handle incomplete types

```

Comment:
It is slightly confusing that this comment isn't on the line to which it refers. Since there isn't really space without egregious wrapping, I suggest:
```suggestion
    : true_type {}; // _STD _Refwrap_ctor_fun is qualified: avoid ADL, handle incomplete types
```